boards/waveshare/esp32s3_matrix/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Joel Guittet
+# Copyright (c) 2025 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 4096 if BOARD_ESP32S3_MATRIX_ESP32S3_PROCPU
+	default 256 if BOARD_ESP32S3_MATRIX_ESP32S3_APPCPU
```

Comment:
think this has been replaced? @marekmatej

Reply:
It's required by `modules/hal/espressif/zephyr/port/heap/heap_caps_zephyr.c`

Reply:
ping @marekmatej

Reply:
@nordicjm Actually, this heap request comes from the boards, so I don't think its wrong.